docs/source/deployment/nodes_grouping.md

```diff
@@ -58,7 +58,7 @@ Namespaces allow you to group nodes, ensuring clear dependencies and separation
 ![Switching expanding namespaced pipeline in Kedro Viz](../meta/images/kedro_viz_expanding_namespace.gif)
 
 Using namespaces comes with a few challenges:
-- **Defining namespace at Node-level:** If you define namespaces at the node level, they behave similarly to tags and do not guarantee execution consistency.
+- **Defining namespace at Node-level:** Defining namespaces at node level is not recommended for grouping your nodes. The node level definition of namespaces should only be used for creating collapsible views on Kedro-Viz for high level representation of your nodes. If you define namespaces at the node level, they behave similarly to tags and do not guarantee execution consistency.
```

Comment:
Hey, I'm thinking that if we don't recommend using it, we could simplify things by moving it to the very top of the section and removing **Defining namespace at Node-level:** entirely.

> Defining namespaces at the node level is not recommended for grouping nodes. Instead, please define your namespace at the pipeline level for deployment.

Reply:
I was thinking it's fine because this section lists the "challenges"

Reply:
yeah but I also feel like if we don't recommend using it then it should be highlight immediately at the start of the section? Rather than near the end. 🤔

Reply:
Updated